jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerContextGenerator.java

```diff
@@ -141,56 +185,26 @@ public void generate(Path targetDirectory) throws IOException {
   }
 
   /**
-   * Makes a {@code Dockerfile} from the {@code DockerfileTemplate}.
-   *
-   * @return the {@code Dockerfile} contents.
-   * @throws IOException if reading the Dockerfile template fails.
-   */
-  @VisibleForTesting
-  String makeDockerfile() throws IOException {
-    Preconditions.checkNotNull(baseImage);
-
-    String dockerfileTemplate =
-        Resources.toString(Resources.getResource("DockerfileTemplate"), StandardCharsets.UTF_8);
-
-    return dockerfileTemplate
-        .replace("@@BASE_IMAGE@@", baseImage)
-        .replace(
-            "@@DEPENDENCIES_PATH_ON_IMAGE@@", sourceFilesConfiguration.getDependenciesPathOnImage())
-        .replace("@@RESOURCES_PATH_ON_IMAGE@@", sourceFilesConfiguration.getResourcesPathOnImage())
-        .replace("@@CLASSES_PATH_ON_IMAGE@@", sourceFilesConfiguration.getClassesPathOnImage())
-        .replace(
-            "@@ENTRYPOINT@@",
-            joinAsJsonArray(
-                EntrypointBuilder.makeEntrypoint(sourceFilesConfiguration, jvmFlags, mainClass)))
-        .replace("@@CMD@@", joinAsJsonArray(javaArguments));
-  }
-
-  /**
-   * Formats a list for the Dockerfile's ENTRYPOINT or CMD.
+   * Makes the contents of a {@code Dockerfile} using configuration data.
```

Comment:
Since we're putting this in code, would be good to have in the javadoc an example of what it would look like.